### PR TITLE
changed some lines

### PR DESCRIPTION
**Pull request by Watson669**
Changed 2 conditions
removed 2 lines for thirsty

### CODE REVIEW

File: hungry.py [hunk 1]
```diff
@@ -1,12 +1,11 @@
 hungry=input("Are you hungry ? ")
-if(hungry.lower()=='yes' or hungry=='yea bro'):
+if(hungry.lower()=='yes' or hungry="eurus" or hungry="sausage"):
```
Comment (Sherlock669) [inline hungry.py]: Why eurus here?

File: hungry.py [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
   print("Eat fries!")
   print("Eat burger!")
   print("Eat pizza!")
 else:
   thirsty = input("Are you thirsty ? ")
   if (thirsty.lower() == 'yes'):
-    print("Drink water")
-    print("Drink soda")
-    print("Drink juice")
+    print("Drink Beer")
```
Comment (Sherlock669) [inline hungry.py]: Bad for health

File: hungry.py [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
-if(hungry.lower()=='yes' or hungry=='yea bro'):
+if(hungry.lower()=='yes' or hungry="eurus" or hungry="sausage"):
   print("Eat KFC!")
   print("Eat fries!")
   print("Eat burger!")
   print("Eat pizza!")
 else:
   thirsty = input("Are you thirsty ? ")
   if (thirsty.lower() == 'yes'):
-    print("Drink water")
```
Comment (Sherlock669) [inline hungry.py:10]: Add "lot of water " here

Comment (Watson669) [reply hungry.py:10]: Okay will do

File: hungry.py (hunk 1 above)
Comment (Watson669) [reply hungry.py]: Simply!

**Pull request merged**